Pass4Win/config.cs

```diff
@@ -44,6 +44,7 @@ public FrmConfig()
                 FrmMain.Cfg["GitUser"] = "";
                 FrmMain.Cfg["GitPass"] = "";
                 FrmMain.Cfg["GitRemote"] = "";
+                FrmMain.Cfg["FirstRun"] = false;
```

Comment:
setting FirstRun to false after the app loads for the first time. This way, it will populate the saved config settings in the config window the next time the app runs